product_profile/README.rst

```diff
@@ -0,0 +1,128 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product Profile
+===============
+
+This module was written to make products configuration easier (in one click).
+It allows to configure a product template with only one field.
+
+.. image:: static/description/field.png
+
+**Main use case**: a lot of modules are installed (mrp, purchase, sale, pos)
+and products configuration becomes harder for end users: too many fields to take care of.
+
+You are concerned that at any time a product might be not configured correctly: this module is your friend.
+
+In this case a lot of complexity becomes hidden (default behavior) to the end user and usability is optimal.
+
+It also can make ease the data migration by only specifying the profile field instead of all fields which depend on it.
+
+Note: This module is meant to be used by skilled people in database fields creation with the ERP framework.
```

Comment:
This module is meant to be used by skilled people in database fields creation within the ERP framework.